Source/embeddings/Builder/EmbeddingClassBuilder.ts

```diff
@@ -27,7 +27,7 @@ import { ICanBuildAndRegisterAnEmbedding } from './ICanBuildAndRegisterAnEmbeddi
 import { OnDecoratedEmbeddingMethod } from './OnDecoratedEmbeddingMethod';
 import { RemoveDecoratedMethod } from './RemoveDecoratedMethod';
 import { RemoveDecoratedMethods } from './RemoveDecoratedMethods';
-import { remove as removeDecorator } from './removeDecorator';
+import { resolveDeletionToEvents as removeDecorator } from './removeDecorator';
```

Comment:
```suggestion
import { resolveDeletionToEvents as deleteDecorator } from './deleteDecorator';
```

JS imports sadly don't rename themselves even if the filename changes. It's hellish fixing all of the `index.ts` files too

Reply:
Done.